packages/t/typescript-vfs.json

```
@@ -0,0 +1,32 @@
+{
+  "description": "A Map based TypeScript Virtual File System.",
+  "filename": "dist/vfs.globals.js",
```

```suggestion
  "filename": "vfs.globals.js",
```
As you've set basePath to dist, you don't need to include it here